### PR TITLE
chore: split the ci into parts

### PR DESCRIPTION
**Pull request by Caele**
## Motivation

Splits the CI into jobs to make it faster (hopefully) and easier to view

### CODE REVIEW

File: .circleci/config.yml
Comment (T-Wizard) [inline .circleci/config.yml:1]: maybe also a install dependencies job (checkout & yarn)
that build and api-governance can depend on (to remove duplicated work)

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,5 +1,10 @@
 version: 2.1
 
+defaults: &defaults
```
Comment (T-Wizard) [inline .circleci/config.yml:3]: where is this used? (or is it not?)

File: .circleci/config.yml
Comment (Caele) [reply .circleci/config.yml:1]: I did this initially, but ran into issues, maybe after i've resolved other things I can put it back in.
Not sure the api-gov actually needs the deps installed actually, i'll see.

File: .circleci/config.yml [hunk 2]
```diff
@@ -72,34 +77,21 @@ commands:
...
             <<# parameters.build >> yarn run build <</ parameters.build >>
             <<# parameters.test >>  yarn run test:e2e <</ parameters.test >>
-
 jobs:
-  build:
+  validate:
     docker:
-      - image: cimg/node:20.2.0
+      - image: cimg/node:20.1.0
       - image: browserless/chrome:1-puppeteer-7.1.0
```
[the remark this answers is not in the document]
Comment (Caele) [reply .circleci/config.yml:84]: hmm, could likely use node-browsers instead, i'll try

File: .circleci/config.yml
Comment (T-Wizard) [reply .circleci/config.yml:1]: ok

**Review by T-Wizard — APPROVED** [review]

**Pull request merged**